exercises/exercise-02/index.ts

```diff
@@ -64,8 +64,8 @@ const persons: Person[] = [
 ];
 
 function logPerson(person: Person) {
-    let additionalInformation: string;
```

Comment:
It's interesting to note that this annotation is not needed. See [this post](https://effectivetypescript.com/2020/03/09/evolving-any/) for details.